diff --git a/epan/dissectors/CMakeLists.txt b/epan/dissectors/CMakeLists.txt
index 0393f263cc..fc215400d5 100644
--- a/epan/dissectors/CMakeLists.txt
+++ b/epan/dissectors/CMakeLists.txt
@@ -1262,6 +1262,7 @@ set(DISSECTOR_SRC
        ${CMAKE_CURRENT_SOURCE_DIR}/packet-h263.c
        ${CMAKE_CURRENT_SOURCE_DIR}/packet-h263p.c
        ${CMAKE_CURRENT_SOURCE_DIR}/packet-h264.c
+       ${CMAKE_CURRENT_SOURCE_DIR}/packet-happp.c
        ${CMAKE_CURRENT_SOURCE_DIR}/packet-h265.c
        ${CMAKE_CURRENT_SOURCE_DIR}/packet-hartip.c
        ${CMAKE_CURRENT_SOURCE_DIR}/packet-hazelcast.c
